libs/api/request-access/api/feature/src/lib/api-request-access.resolver.ts

```diff
@@ -6,9 +6,16 @@ import { ApiRequestAccessService } from "./api-request-access.service";
 export class ApiRequestAccessResolver {
     constructor(private requestAccessService: ApiRequestAccessService) {}
 
-    @Mutation(returns => ApiRequestAccessEntity)
+    @Mutation(returns => ApiRequestAccessEntity, { nullable: true })
     // must add paramters to request
-    async requestAccess(@Args('compID', { type: () => ID }) compId: string, @Args('gradID', { type: () => ID }) grad: string, @Args('item') id: string): Promise<ApiRequestAccessEntity> {
+    async requestAccess(@Args('compId', { type: () => ID }) compId: string, @Args('gradId', { type: () => ID }) gradId: string, @Args('item') item: string): Promise<ApiRequestAccessEntity> {
+        if (compId == "" || gradId == "" || item == "") // obviously empty elements are not allowed
+            return null;
+
+        // if new items become available add here
+        if (!["CV", "Transcript", "Academic", "Certificates", "Capstone"].includes(item)) // if a valid item has been requested
```

Comment:
```suggestion
        if (!["CV", "Transcript", "Academic", "Certificates", "Capstone"].includes(item)) // if an invalid item has been requested
```
Null is returned if the request is invalid